.dorpsgek.yml

```diff
@@ -0,0 +1,15 @@
+notifications:
+  global:
+    irc:
+      - openttd
+      - openttd.notice
+
+  push:
+    only:
+    - master
+    only-by:
+    - DorpsGek
```

Comment:
please remove the `push` section, it is only useful for OpenTTD main project (DorpsGek will never make a commit in this repo)